docs/source/starter/installation.rst

```diff
@@ -67,16 +67,20 @@ If desired, the GeoIPS environment variables can be added to your
       # wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
       wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
 
-- Make the install script executable and run the installer,
-  following the prompts (particularly the bit about
-  conda init / restarting terminal!):
+- Make the installation script executable, and run the installer.
+  Accept the license terms and cofirm the location.
+- You would see a message from installer asking to close and
```

Comment:
"You would see a message from installer asking to close and re-open the shell at the end of installation which is not required."

This has grammatical and clarity issues.
- "would" vs "will" - incorrect tense
- missing a subject after asking; though not required, it makes the sentence clearer.
- "from installer" should be "from the installer" 
- "asking to close and re-open the shell" is awkward. adding "you" after "asking" specifies who is being asked to do this and improves sentence flow
- "which is not required" at the end of the sentence is unclear because it doesn't specify what exactly is "not required." meaning can be inferred, but it can also be made more explicit

Suggested change:

"You will see a message from the installer asking you to close and re-open the shell at the end of the installation, but this step is not required."

Reply:
I aimed to convey a sense of certainty by using "would" instead of "will". The reviewed line is replaced with the above suggested statement. https://github.com/NRLMMD-GEOIPS/geoips/commit/e944c71a899963495a9af2ab531887610bd78695